### PR TITLE
[Serializer] Minor: fix CS and PHPDoc

### PR DESCRIPTION
**Pull request by dunglas**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | n/a |
| License | MIT |
| Doc PR | n/a |

### CODE REVIEW

File: src/Symfony/Component/Serializer/Mapping/ClassMetadataInterface.php [hunk 1]
```diff
@@ -41,7 +41,7 @@ public function addAttributeMetadata(AttributeMetadataInterface $attributeMetada
     /**
      * Gets the list of {@link AttributeMetadataInterface}.
      *
-     * @return AttributeMetadataInterface[]
+     * @return array An array of {@see AttributeMetadataInterface} instances.
```
Comment (xabbuh) [inline src/Symfony/Component/Serializer/Mapping/ClassMetadataInterface.php]: Do the keys convey a special meaning? Or what's the advantage over using `AttributeMetadataInterface[]`?

Comment (dunglas) [inline src/Symfony/Component/Serializer/Mapping/ClassMetadataInterface.php]: Keys are string (the attribute name). Static analysis tools including Scrutinizr use the following convention: `[]` for packed arrays and simply `array` for associative arrays.

Here using `[]` prevent such tools of working properly.

Comment (xabbuh) [inline src/Symfony/Component/Serializer/Mapping/ClassMetadataInterface.php]: Then we should describe the meaning of keys in the docblock imo.

File: src/Symfony/Component/Serializer/Serializer.php [hunk 2]
```diff
@@ -281,7 +285,6 @@ private function denormalizeObject($data, $class, $format, array $context = arra
         foreach ($this->normalizers as $normalizer) {
             if ($normalizer instanceof DenormalizerInterface
                 && $normalizer->supportsDenormalization($data, $class, $format)) {
-
                 return $normalizer->denormalize($data, $class, $format, $context);
```
Comment (jvasseur) [inline src/Symfony/Component/Serializer/Serializer.php]: This align the `&&` with the `return`, reducing readability. Maybe the if should be put on one line.

**Comment by fabpot** [comment]
As we are using this convention everywhere in the codebase, I'm :-1: to change this just here.

**Comment by dunglas** [comment]
I reverted PHPDoc changes and only kept minor CS fixes.

**Comment by fabpot** [comment]
Thank you @dunglas.

**Pull request closed without merging**